### DISCUSSION TITLE
plans for BITMAP support

### DISCUSSION THREAD

**Discussion by koron** [dragonflydb/dragonfly#212] [Q&A] [answered]
Are there any plans to support the [BITMAP commands group](https://redis.io/commands/?group=bitmap)?

**Comment by romange** [comment, marked as answer]
yes, we plan to.
could you describe how you use it?

**Reply by koron** [reply]
thank you to answer.

i'm using `bitfield` command to implement specialized bloom filter with transaction (`multi`)

* https://github.com/koron/java-vbf/blob/b007e91e63df72924f09ffbc085faa24381f19ae/src/main/java/net/kaoriya/vbf3/RedisVBF3.java#L291
* https://github.com/koron/java-vbf/blob/b007e91e63df72924f09ffbc085faa24381f19ae/src/main/java/net/kaoriya/vbf3/RedisVBF3.java#L264

**Reply by romange** [reply]
look cool.  do you mind opening an issue for this? if you use only the subset of bitops commands, please specify what you need to raise its priority. And if your codebase happens to have an integration test that works with Redis, I would appreciate, if you detail how you run it against redis endpoint.

**Reply by koron** [reply]
I wrote an issue: https://github.com/dragonflydb/dragonfly/issues/213

Please let me know if you need any additional information. I will add it.

**Reply by romange** [reply]
Thanks, Muraoka. Just to manage the expectations - most likely, it won't be done in Q3. I will add it to Q4 plan.
